.github/workflows/tests.yml

```diff
@@ -0,0 +1,41 @@
+name: Tests
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - 'docs/**'
+      - '*.rst'
+  pull_request:
+    branches:
+      - master
```

Comment:
Would we not want to have tests fire on _all_ pull requests? Ie, if someone does a stacked PR, be nice to have tests fire.